cli/pcluster/commands.py

```diff
@@ -38,11 +38,9 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def _create_bucket_with_resources(pcluster_config, json_params, tags):
+def _create_bucket_with_resources(pcluster_config, storage_data, tags):
     """Create a bucket associated to the given stack and upload specified resources."""
     scheduler = pcluster_config.get_section("cluster").get_param_value("scheduler")
-    if scheduler not in ["awsbatch", "slurm"]:
-        return None
```

Comment:
Because this check is gone, i think this function is uploading data that it doesn't need to for SGE and torque clusters. We should avoid doing that for those.

Reply:
Fixed